docusaurus/docs/testing/e2e-test.md

```diff
@@ -200,3 +200,26 @@ Both unit and E2E tests generate coverage respectively with Jest and NYC. These
 Special attention goes to the E2E as the code is instrumented with Babel and the configuration is set within Nuxt.js.
 
 >  Note: To enable instrumentation required for code coverage, you must set the environment variable `TEST_INSTRUMENT` to `true`.
+
+
+## Cypress Tags (cypress-grep plugin)
+
+Cypress-grep is a plugin that was implemented into our project to add the ability to group our e2e tests and we do that by adding tags to the tests.
+
+Reasons we are grouping our tests:
+- Introduction of standard user to our workflow to add test coverage for that user type (currently we're only testing against admin user). 
+- In the near future we will add tags for CI - quicker test runs and overnight/scheduled - longer test runs which will reduce the time it takes for our tests to run when making a commit or creating a pr.
```

Comment:
--> In the future support sets of tests specific to the context they run in (PR CI, overnight, etc)